### PR TITLE
Add test cases and example of monitor solutions.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Add test cases and example of monitor solutions.

### CODE REVIEW

File: src/MonitoringSolutions/docs/New-AzMonitorLogAnalyticsSolution.md [hunk 1]
```diff
@@ -23,23 +23,17 @@ Creates a log analytics solution.
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Create a monitor log analytics solution for the log analytics workspace
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> $workspace = Get-AzOperationalInsightsWorkspace -ResourceGroupName azureps-manual-test -Name monitoringworkspace-2vob7n
```
Comment (isra-fel) [inline src/MonitoringSolutions/docs/New-AzMonitorLogAnalyticsSolution.md:28]: We can add the link to the help document of `Get-AzOperationalInsightsWorkspace` in the "Related Links" section, at the bottom of this doc.
```markdown
## RELATED LINKS

[Get-AzOperationalInsightsWorkspace](https://docs.microsoft.com/en-us/powershell/module/az.operationalinsights/get-azoperationalinsightsworkspace)
```

File: src/MonitoringSolutions/docs/Get-AzMonitorLogAnalyticsSolution.md [hunk 2]
```diff
@@ -41,23 +41,50 @@ Retrieves the user solution.
...
 ```
 
-{{ Add description here }}
+This command gets a monitor log analytics solution by name.
 
-### Example 2: {{ Add title here }}
+### Example 2: Get a monitor log analytics solution by object
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> $monitor = Get-AzMonitorLogAnalyticsSolution -ResourceGroupName azureps-monitor -Name 'Containers(azureps-monitor)'
```
Comment (isra-fel) [inline src/MonitoringSolutions/docs/Get-AzMonitorLogAnalyticsSolution.md]: I think the correct way to call `GetViaIdentity` is to:

1. constrcut a hashtable containing the identity information of the resource
1. pass the hashtable to `GetViaIdentity` to get the full resource

For example:

```powershell
@{Id = "/subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f/resourcegroups/yemingmonitor/providers/microsoft.operationalinsights/workspaces/yemingmonitor"} | Get-AzMonitorLogAnalyticsSolution
```

File: src/MonitoringSolutions/docs/Remove-AzMonitorLogAnalyticsSolution.md [hunk 3]
```diff
@@ -29,23 +29,22 @@ Deletes the solution in the subscription.
...
 
-{{ Add description here }}
+This command removes a monitor log analytics solution by name.
 
-### Example 2: {{ Add title here }}
+### Example 2: Remove a monitor log analytics solution by object
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> $monitor = Get-AzMonitorLogAnalyticsSolution -ResourceGroupName azureps-manual-test -Name 'Containers(monitoringworkspace-pevful)'
+PS C:\> Remove-AzMonitorLogAnalyticsSolution -InputObject $monitor
```
Comment (isra-fel) [inline src/MonitoringSolutions/docs/Remove-AzMonitorLogAnalyticsSolution.md:43]:
```suggestion
PS C:\> $monitor | Remove-AzMonitorLogAnalyticsSolution
```
We can sometimes write examples using pipeline syntax [more info](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_pipelines?view=powershell-7)

It's recommended and easier to use

File: src/MonitoringSolutions/test/test-new.ps1 [hunk 4]
```diff
@@ -6,9 +6,10 @@
 
 $workspace = Get-AzOperationalInsightsWorkspace -ResourceGroupName yemingmonitor -Name yemingmonitor
```
Comment (isra-fel) [inline src/MonitoringSolutions/test/test-new.ps1]: I will remove this file later

**Review by isra-fel — CHANGES_REQUESTED** [review]
Please see my inline comments.
Thanks!

File: src/MonitoringSolutions/docs/New-AzMonitorLogAnalyticsSolution.md (hunk 1 above)
Comment (LucasYao93) [reply src/MonitoringSolutions/docs/New-AzMonitorLogAnalyticsSolution.md:28]: Added releated links about Get-AzOperationalInsightsWorkspace.

File: src/MonitoringSolutions/docs/Get-AzMonitorLogAnalyticsSolution.md (hunk 2 above)
Comment (LucasYao93) [reply src/MonitoringSolutions/docs/Get-AzMonitorLogAnalyticsSolution.md]: Add a new example description for this.

File: src/MonitoringSolutions/docs/Remove-AzMonitorLogAnalyticsSolution.md (hunk 3 above)
Comment (LucasYao93) [reply src/MonitoringSolutions/docs/Remove-AzMonitorLogAnalyticsSolution.md:43]: Add a new example description for this.

File: src/MonitoringSolutions/test/test-new.ps1 (hunk 4 above)
Comment (LucasYao93) [reply src/MonitoringSolutions/test/test-new.ps1]: Deleted this file.

File: src/MonitoringSolutions/docs/New-AzMonitorLogAnalyticsSolution.md (hunk 1 above)
Comment (LucasYao93) [reply src/MonitoringSolutions/docs/New-AzMonitorLogAnalyticsSolution.md:28]: Added tag LINK in custom/New-AzMonitorLogAnalyticsSolution.ps1 and rebuilded.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**